yaspin/core.py

```diff
@@ -444,9 +444,11 @@ def _compose_out(self, frame: str, mode: Optional[str] = None) -> str:
         if self._side == "right":
             frame, text = text, frame
         # Timer
-        if self._timer:
+        if self._timer is not False:
+            # TODO the format string should allow showing custom groupings (such as total seconds or HH:MM:SS)
+            timer_format = self._timer if isinstance(self._timer, str) else " ({}.{:02.0f})"
             sec, fsec = divmod(round(100 * self.elapsed_time), 100)
-            text += " ({}.{:02.0f})".format(  # pylint: disable=consider-using-f-string
+            text += timer_format.format(  # pylint: disable=consider-using-f-string
```

Comment:
What if `timer_format` has no slots or only one slot (while `.format()` is called with 2 arguments) or more than 2 slots?